config/config.yaml

```diff
@@ -90,6 +85,10 @@ multiqc:
         - "qc/mosdepth_bed/{sample}_{type}.mosdepth.region.dist.txt"
         - "qc/mosdepth_bed/{sample}_{type}.mosdepth.global.dist.txt"
 
+pbrun_deepvariant:
```

Comment:
If using parabricks 4.0.0 then the  deepvariant model files needs for an A30 GPU needs to be specified.
https://docs.nvidia.com/clara/parabricks/4.0.0/documentation/tooldocs/man_deepvariant.html#models-for-additional-gpus

For example on our A30 GPUs, we need to specify a path the a WES model for A30 GPUs:
extra: "--pb-model-file /beegfs-storage/data/parabricks/4.0.0-1/A30/wes/deepvariant_wes.eng 

But if you use parabricks 4.1.0 as this does not need to be specified  ( as they ship with the container) and you can run it as is. So most straightforward to run with parabricks 4.1.0, which also runs the latest deepvariant 1.5